todo_cli/cli/cli_controller.py

```diff
@@ -1,8 +1,11 @@
 """
 This module contains all logic pertaining to the CLI commands
 """
+import typer
 from todo_cli.helpers.tasks_helper import read_tasks_file, create_task, write_tasks_file, view_tasks
 
+app = typer.Typer()
+
 def handle_cli_args(cli_args: list[str]) -> None:
```

Comment:
This function is no longer required, Typer now handles our commands.

Reply:
I am referring to the handle_cli_args function as github is giving a weird snippet here